_announcements/2020-03-25-announcement-of-emergency-board-meeting.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Announcement of Emergency Board Meeting

```

Comment:
I like the consistent use of the headings and brief explanatory sentence or two at the top of each page, as well as the date at the top of each page. I was hoping for additional navigation within the Announcements section, so I could navigate between the featured announcements without having to return to "Announcements" in the left nav bar each time I wanted to read a new announcement.

Reply:
Good feedback. That's unfortunately a slightly heavy lift. I can look at adding it in the future though!